>From 4e317d4c4bf00c59e47f88b17b6583ae01cf56df Mon Sep 17 00:00:00 2001
From: Robbin Edgren <robbin@edgren.pp.se>
Date: Mon, 29 Sep 2014 09:14:19 +0200
Subject: [PATCH] Judaic calendars must conform to calendar format.

The man page of calendar states:

	Day descriptions start after the first <tab>
	character in the line; if the line does not
	contain a <tab> character, it isn't printed
	out.

However, the output of calendarJudaic.py doesn't have a
tabulator but two spaces instead. This patch fixes that.
---
 debian/calendarJudaic.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/debian/calendarJudaic.py b/debian/calendarJudaic.py
index 91f4bfe..d4c9639 100644
--- a/debian/calendarJudaic.py
+++ b/debian/calendarJudaic.py
@@ -127,23 +127,23 @@ LANG=UTF-8
     parasha['diaspora'] = h.get_parasha_string(0)
     if holyday['Israel']:
       if holyday['Israel'] == holyday['diaspora']:
-        print "%02d/%02d*  %s" % (day, month, holyday['Israel'])
+        print "%02d/%02d*	%s" % (day, month, holyday['Israel'])
       else:
-        print "%02d/%02d*  %s (Israel only)" % \
+        print "%02d/%02d*	%s (Israel only)" % \
                                 (day, month, holyday['Israel'])
     elif holyday['diaspora']:
-      print "%02d/%02d*  %s (diaspora only)" % \
+      print "%02d/%02d*	%s (diaspora only)" % \
                                 (day, month, holyday['diaspora'])
     if parasha['Israel']  and  parasha['Israel'] != 'none':
       if parasha['Israel'] == parasha['diaspora']:
-      	print "%02d/%02d*  Parshat %s" % \
+        print "%02d/%02d*	Parshat %s" % \
                                 (day, month, parasha['Israel'])
       else:
-        print "%02d/%02d*  %s (Israel only)" % \
+        print "%02d/%02d*	%s (Israel only)" % \
                                 (day, month, parasha['Israel'])
     elif parasha['diaspora']  and  parasha['diaspora'] != 'none':
     # See http://bugs.debian.org/583092 why 'none' is required here.
-      print "%02d/%02d*  Parshat %s (diaspora only)" % \
+      print "%02d/%02d*	Parshat %s (diaspora only)" % \
                                 (day, month, parasha['diaspora'])
 
   print footer
-- 
1.9.1

